R/vdiagram-data.R

```diff
@@ -70,11 +70,8 @@ createControlConditionsString <- function(controlConditions){
   returnString <- ""
   
   if(!isEmptyOrBlank(controlConditions)){
-    displayValues <- controlConditions[['name']]
-    
```

Comment:
hopefully they don't care about this, but if they do, we'll have to get the prettier name back in there maybe

Reply:
actually maybe I can modify this to split on the capital letter ...